kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/ProcessServiceIntegrationTest.java

```diff
@@ -486,10 +486,6 @@ public void testWorkItemOperations() throws Exception {
 
             processClient.abortWorkItem(CONTAINER_ID, processInstanceId, workItemInstance.getId());
 
-            workItems = processClient.getWorkItemByProcessInstance(CONTAINER_ID, processInstanceId);
```

Comment:
this was removed as it now throws process instance not found exception since the instance is completed after workitem was aborted. So instead we can check if the process exists. wdyt?